File: Python/tests/test_bandit4arm_4par.py
import pytest

from hbayesdm.models import bandit4arm_4par


def test_bandit4arm_4par():
    _ = bandit4arm_4par(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bandit4arm_4par.R
context("Test bandit4arm_4par")
library(hBayesDM)

test_that("Test bandit4arm_4par", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bandit4arm_4par(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
